packages/sshnoports/scripts/install_sshnpd

```diff
@@ -35,6 +35,11 @@ usage() {
   echo "  -d, --device <address>      Device address (e.g. @alice_device)"
   echo "  -n, --name <device name>    Name of the device"
   echo "  -v, --version <version>     Version to install (default: latest)"
+  echo "      --args <args>           Additional arguments to sshnpd ("-v" by default)"
+  echo "      Possible args:"
+  echo "        -s, --[no-]sshpublickey      Update authorized_keys to include public key from sshnp"
+  echo "        -u, --[no-]username          Send username to the manager to allow sshnp to display username in command line"
```

Comment:
Please use the new description associated with the -u which is now --un-hide (although still also aliased to --username for back compatibility

Reply:
From sshnpd params:

parser.addFlag(
      'un-hide',
      abbr: 'u',
      aliases: const ['username'],
      defaultsTo: false,
      help:
          'When set, makes various information visible to the manager atSign - e.g. username, version, etc',
    );